internal/consumer/service.go

```diff
@@ -0,0 +1,81 @@
+package consumer
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/emmaLP/gs-software-onboarding/internal/model"
```

Comment:
Imports should be grouped ([see more](https://github.com/gymshark/software-engineering-standards/blob/main/languages/go/style-guide.md#import-group-ordering)). 

[See here for setting up an editor to do it automagically](https://github.com/gymshark/software-engineering-standards/blob/main/languages/go/style-guide.md#linting).

Reply:
Not sure what you mean by this comment as my imports are grouped according to the linting and the imports standards. Please can you elaborate on your comment

Reply:
Interestingly when I import local dependencies (e.g. `github.com/emmaLP/gs-software-onboarding/internal/model`) it adds it as a separate group to the external dependencies. Maybe it's something to do with my local config.

Reply:
Hmm, that is interesting. I'll resolve this convo with no action as it following the engineering standards guide